myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java

```diff
@@ -197,6 +197,8 @@
   @JsonProperty
   private String cgroupPath;
 
+  @JsonProperty
+  private Boolean cgroupEnabled;
```

Comment:
Small nit, recommend defining default value for this and initializing it accordingly